scss/color.scss

```diff
@@ -0,0 +1 @@
+$color-black: #000000;
```

Comment:
This is a detail, but you can just write `#000` and will work the same. It works the same with repeated hexs, like `#FFF`